arviz/plots/backends/bokeh/pairplot.py

```diff
@@ -56,6 +58,36 @@ def plot_pair(
         kde_kwargs["contour_kwargs"].setdefault("line_color", "black")
         kde_kwargs["contour_kwargs"].setdefault("line_alpha", 1)
 
+    if true_values:
+        true_values_copy = {}
+        label = []
+        for variable in list(true_values.keys()):
+            if " " in variable:
```

Comment:
Hi! I was trying to figure out a way so that the user can specify variable names in an easy manner. For example,  simply writing "theta Choate"  instead of "theta\nChoate"